velox/experimental/breeze/test/generated/algorithms/algorithm_test-openmp.h

```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// Copyright (c) 2024 by Rivos Inc.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
```

Comment:
Now that we're checking in this code. Let's make the generator code add a comment here that makes it clear that this file was auto-generated and should not be edited directly. Something like:

```
/*
 * This file is auto-generated from test/generate.sh
 * DO NOT EDIT!
 */
```

Reply:
Sorry only saw this comment after merging, it's a good idea, can you add it to #11537?